packages/styles/scss/components/layout/_carousel-view.scss

```diff
@@ -0,0 +1,45 @@
+.carousel {
+  --carousel-is-vertical: 1;
+  --carousel-is-horizontal: 0;
```

Comment:
these are mutually exclusive. would be nicer if we could represent them as a single var. 
thinking whether it might be more in line with our other settings if this was in fact a data attribute?

Reply:
Because it is CSS, inverting is not a simple operation. But now I derive the horizontal value from the vertical and only have to set `--carousel-is-vertical`